src/actions/formActions.js

```diff
@@ -10,24 +12,36 @@ export const formActions = {
   TOGGLE_SHOW_FORM,
 };
 
-export const toggleShowForm = formName => ({
-  type: TOGGLE_SHOW_FORM,
-  formName,
-});
+export const addChip = (router, name, input, limit) => {
```

Comment:
Not a fan of tightly coupling all these components to react router if we can avoid it. Keep in mind they have completely rewritten their api 2 times already. I also don't think these would belong in actions since these are just functions, not actions.